bessctl/conf/testing/module_tests/exact_match.py

```diff
@@ -0,0 +1,45 @@
+# Crash test -- generate a bunch of rules and stuff packets through
+em0 = ExactMatch(fields=[{'offset': 23, 'size': 1},  # random fields, I have no idea what these are
+                         {'offset': 2, 'size': 2},
+                         {'offset': 29, 'size': 1}])
+
+em0.add(fields=[chr(255), chr(0) + chr(12), chr(55)], gate=0)
```

Comment:
Not a big deal, but `[b'\xff', b'\x00\x0c\x37']` would be more representative. After all, `bytes` should be preferred than `str`.